cmd/sftp-server.go

```diff
@@ -428,7 +437,7 @@ func startSFTPServer(args []string) {
 			allowMACs = filterAlgos(arg, strings.Split(tokens[1], ","), supportedMACs)
 		case "trusted-user-ca-key":
 			userCaKeyFile = tokens[1]
-		case "password-auth":
+		case "disable-password-auth":
```

Comment:
This is to match the doc https://min.io/docs/minio/linux/reference/minio-server/minio-server.html#minio.server.-sftp